### PR TITLE
fix: Use environment var instead of production

### PR DESCRIPTION
**Pull request by Crash--**

### CODE REVIEW

**Comment by Crash--** [comment]
[Mise à jour webpack - l'erreur "redux en production" sur mobile](https://trello.com/c/jrAsrY7F/1515-mise-%C3%A0-jour-webpack-lerreur-redux-en-production-sur-mobile)

**Comment by CPatchane** [comment]
LGTM 👍 
@Crash-- You have to update the test snapshots as well, the build is broken.

File: packages/cozy-scripts/config/webpack.target.mobile.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const webpack = require('webpack')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin')
 
-const { production, isDebugMode } = require('./webpack.vars')
+const { environment, isDebugMode } = require('./webpack.vars')
 const manifest = fs.readJsonSync(paths.appManifest())
```
Comment (CPatchane) [inline packages/cozy-scripts/config/webpack.target.mobile.js]: Actually, you want to have the `production` variable here like

```suggestion
 const production = environment === 'production
```

**Comment by CPatchane** [comment]
Thanks @Crash-- 👍

**Pull request merged**